### DISCUSSION TITLE
Getting Eye controller placement where I want it

### DISCUSSION THREAD

**Discussion by benngarnish** [jjburton/cgmTools#226] [Ideas]
How am I able to get the eye placement to be closer to the center of the face, further out from the snout. 
Maybe using the eyeMain block with some options to visualise where the eye controller will be created and move some object in the scene to get the controller showing up where I want it to be.

![eyeLocs](https://user-images.githubusercontent.com/1122587/115247782-fff0e400-a11e-11eb-87a7-8694eaedf3fb.png)

**Comment by jjburton** [comment]
There's a couple of ways to tackle it. One option would be to simply place the end aimers on the same plane as the eyeLook and the aim constraint would have an offset so they wouldn't be on the aim vector of the eye itself.
![image](https://user-images.githubusercontent.com/48218358/115247649-9821a580-a0ec-11eb-9a8a-6379ffccb382.png)
